### PR TITLE
Add Baidosters00d Kernersville NC file

### LINKED ISSUE #5: create a file for baidosters00d Kernersville NC
- [x] Create a branch
- [ ] add comments
- [ ] open a pull request
- [ ] collaborate
- [ ] recieve approval :+1: 
- [ ] merge :shipit:

### PR DESCRIPTION
**Pull request by baidosters00d**
closes #5

### CODE REVIEW

File: baidosters00d-kernersvillenc.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Kernersville, NC
+
+#### Places to Go
+
```
Comment (YashSV) [inline baidosters00d-kernersvillenc.md:7]: Do you know any other places :)

Comment (baidosters00d) [reply baidosters00d-kernersvillenc.md:7]: We hang out at the Library

**Comment by baidosters00d** [comment]
@cregister1969 does this look good to :shipit: ?

**Comment by YashSV** [comment]
@baidosters00d Pull request ready to :ship: ?

**Comment by YashSV** [comment]
:+1:

**Comment by cregister1969** [comment]
Looks good :shipit:

**Comment by baidosters00d** [comment]
> @cregister1969 does this look good to  ?

:shipit:

**Comment by cregister1969** [comment]
sfd sdf sf sf sdf 

> We hang out at the Library

@baidosters00d  is it far away?

**Pull request merged**